src/watch/mod.rs

```diff
@@ -162,7 +131,8 @@ impl Watch {
         tunnel
             .req_sender
             .send(WatchRequest::create(key_range))
-            .expect("emit watch request");
+            .await
+            .unwrap_or_else(|_| panic!("emit watch request"));
```

Comment:
`unwrap_or_else(|e|`
do not skip error handling

Reply:
do not close unresolved comments

Reply:
Add error message here